diff -burNp ypbind-mt-1.22/src/ypbind_dbus_nm.c ypbind-mt-1.22-dud/src/ypbind_dbus_nm.c
--- ypbind-mt-1.22/src/ypbind_dbus_nm.c	2007-02-14 14:01:57.000000000 +0100
+++ ypbind-mt-1.22-dud/src/ypbind_dbus_nm.c	2009-02-26 14:47:50.072823141 +0100
@@ -50,6 +50,10 @@ typedef enum NMState {
 
 #endif
 
+#ifndef NM_DBUS_SIGNAL_STATE_CHANGE
+#define NM_DBUS_SIGNAL_STATE_CHANGE "StateChange"
+#endif
+
 #include "ypbind.h"
 #include "log_msg.h"
 #include "local.h"
